### PR TITLE
Added Coinbase trading base sync

### PR DESCRIPTION
**Pull request by mortezaalizadeh**

### CODE REVIEW

File: src/exchange/binance-processor/commands/start.go [hunk 1]
```diff
@@ -34,6 +34,8 @@ func startCommand() *cobra.Command {
 				sugarLogger.Fatal(err)
 			}
 
+			binance.UseTestnet = config.Binance.UseTestnet
```
Comment (sepisoad) [inline src/exchange/binance-processor/commands/start.go:37]: why is it hardcoded?

File: src/exchange/coinbase-processor/commands/start.go [hunk 2]
```diff
@@ -32,6 +32,10 @@ func startCommand() *cobra.Command {
 				sugarLogger.Fatal(err)
 			}
 
+			if config.Coinbase.UseSandbox {
```
Comment (sepisoad) [inline src/exchange/coinbase-processor/commands/start.go:35]: again, why hardcoded

**Review by sepisoad — CHANGES_REQUESTED** [review]
awesome, just two minor notes

File: src/exchange/binance-processor/commands/start.go (hunk 1 above)
Comment (mortezaalizadeh) [reply src/exchange/binance-processor/commands/start.go:37]: what do you mean hardcoded? @sepisoad

File: src/exchange/coinbase-processor/commands/start.go (hunk 2 above)
Comment (mortezaalizadeh) [reply src/exchange/coinbase-processor/commands/start.go:35]: what do you mean hardcoded? @sepisoad

Comment (mortezaalizadeh) [reply src/exchange/coinbase-processor/commands/start.go:35]: the values are set in the config file and can be overridden with Environment varaiuble

**Pull request merged**